### ISSUE TITLE
Lang files' loading doesn't work on Android

### ISSUE THREAD

**Issue by RedShakespeare** [CleverRaven/Cataclysm-DDA#40132]
# Describe the bug

In Settings-Interface-Language there are only English/system language two choices.

# Steps To Reproduce

1. Choose Settings and switch to Interface tab.

# Expected behavior

? needless to say

# Screenshots
![Screenshot_2020-05-04-18-24-39-90_6fe8938556442e3d4f710ab4f9a16601](https://user-images.githubusercontent.com/44523519/80957264-68bf4d80-8e35-11ea-9895-0af54d23f4fc.png)

# Versions and configuration

- OS: Android
- Game Version: 0.E-1726-g0c4d9b2(build 10620 and after)
# Additional context
What happened?

**Comment by RedShakespeare** [comment]
.mo files are still there but none of them 
is loaded

**Comment by ZhilkinSerg** [comment]
Probably related to #40056

**Comment by jbytheway** [comment]
Indeed; I guess the logic @akirashirosawa used in #40056 doesn't work on Android.  Is something different about the directory layout there?

**Comment by RedShakespeare** [comment]
> Is something different about the directory layout there?

Oh yes ... I think I may know how to fix it now...thx

**Comment by RedShakespeare** [comment]
No...just underestimated the difficulty. Still need help.

**Comment by RedShakespeare** [comment]
5 days passed but still no fixes...
Many complaints in local social media groups...

**Comment by anothersimulacrum** [comment]
Could it be this:
```


What happens is that PATH_INFO::langdir() is called at include-time, before PATH_INFO::set_standard_filenames() is called. That means that langdir_value = "", which means that the CWD will be scanned instead of the correct lang directory.

This is not really problematic if the current working directory is the game root (i.e. via cataclysm-launcher), but the game used to start fine directly from the binary and I think this is not expected behavior.
```
From #40356

**Comment by RedShakespeare** [comment]
Fixed by #40377 .
Thx a million.
